src/main/java/dinkplugin/notifiers/data/PkNotificationData.java

```diff
@@ -0,0 +1,47 @@
+package dinkplugin.notifiers.data;
```

Comment:
Class & file name are `PkNotification` while the notifier is `PlayerKill..`  - can this be renamed for consistency?